src/student/dto/segment.dto.ts

```diff
@@ -0,0 +1,18 @@
+import { Expose } from "class-transformer";
```

Comment:
This DTO will be reused for student, teacher etc. Call it usersegment.dto and keep it in a common folder instead of under student.